lib/common/patchset.c

```diff
@@ -709,9 +706,7 @@ apply_v2_patchset(xmlNode *xml, const xmlNode *patchset)
                 crm_trace("Adding %s at position %d (first)",
                           child->name, position);
                 CRM_LOG_ASSERT(position == 0);
-                xmlAddChild(match, child);
```

Comment:
Was this line always redundant?

Reply:
No, it became redundant when I replaced `xmlDocCopyNode()` with `pcmk__xml_copy()`. `pcmk__xml_copy()` makes `change->children` the last child of `match`, which is the same thing `xmlAddChild()` does.

Reply:
In fact, I'll double check but I think `xmlAddNextSibling()` became redundant for the same reason

Reply:
yep